vterm.el

```diff
@@ -432,14 +447,34 @@ This is the value of `next-error-function' in Compilation buffers."
     (use-local-map vterm-mode-map)
     (vterm-send-start)))
 
-(defun vterm-copy-mode-done ()
-  "Save the active region to the kill ring and exit `vterm-copy-mode'."
-  (interactive)
+(defun vterm-copy-mode-done (arg)
+  "If a region is active then kill it, otherwise kill the current line, then exit `vterm-copy-mode'.
```

Comment:
It is better if the first sentence of a docstring stays under 67 characters. What you had before is fine (since you explain what the function does after the first sentence).

Reply:
Changed back to the first draft.